test/parallel/test-process-isX-deprecation.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+require('../common');
+const assert = require('assert');
+const { spawnSync } = require('child_process');
+
+[
+  'isArrayBuffer',
+  'isArrayBufferView',
+  'isAsyncFunction',
+  'isDataView',
+  'isDate',
+  'isExternal',
+  'isMap',
+  'isMapIterator',
+  'isNativeError',
+  'isPromise',
+  'isRegExp',
+  'isSet',
+  'isSetIterator',
+  'isTypedArray',
+  'isUint8Array',
+  'isAnyArrayBuffer'
+].forEach((method) => {
+  const { stderr } = spawnSync(process.execPath, ['-p', `
+    process.binding('util')['${method}']('meow');
+  `]);
```

Comment:
Can't this be tested within the same process using `common.expectWarning`? (Probably only for the first time, but that's probably good enough?)

Reply:
We normally use our helper to verify the deprecation messages similar to:

```js
common.expectWarning('DeprecationWarning', {
  DEP0103: message
});
```

Reply:
I don't know why, but it just emit one warning, only the first method.

```js
[ 'isArrayBuffer', ...methods].forEach(method => {
 process.binding('util')[method]('foo')
})
```
`(node:6805) [DEP0103] DeprecationWarning: Accessing native typechecking bindings of Node directly is deprecated. Please use 'util.types.isArrayBuffer' instead.`

That's the reason I used child processes and eval the stderr.

Reply:
It is meant to only trigger a single deprecation warning with the same deprecation number.
I think it's fine to keep the test as it is. It's a really thorough one.